### PR TITLE
Updating Login URL Examples

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

The access URLs will be unique per client/region and embedded links of the `cloud.getdbt.com` URL in our docs need to be updated to the generic YOUR_ACCESS_URL.

URLs that link to a page in the dbt Cloud product, will be redirected to the Regions & IP Address page that will include instructions on how to find a clients URL. 

This does not currently touch API pages, just /docs and /guides that link out to cloud.getdbt.com

## Checklist

- [x] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [x] For [docs versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#about-versioning), review how to [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content).
- [x] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

### CODE REVIEW

File: website/docs/docs/cloud/dbt-cloud-ide/develop-in-the-cloud.md [hunk 1]
```diff
@@ -39,7 +39,7 @@ To improve your experience using dbt Cloud, we suggest that you turn off ad bloc
 
 ## Prerequisites
 
-- A [dbt Cloud account](https://cloud.getdbt.com/) and [Developer seat license](/docs/cloud/manage-access/seats-and-users)
+- A [dbt Cloud account]() and [Developer seat license](/docs/cloud/manage-access/seats-and-users)
```
Comment (mirnawong1) [inline website/docs/docs/cloud/dbt-cloud-ide/develop-in-the-cloud.md]:
```suggestion
- A [dbt Cloud account](https://www.getdbt.com/signup) and [Developer seat license](/docs/cloud/manage-access/seats-and-users)
```

**Comment by mirnawong1** [comment]
hey @matthewshaver , is this pr still being worked on?

File: website/docs/guides/how-to-use-databricks-workflows-to-run-dbt-cloud-jobs.md [hunk 2]
```diff
@@ -130,8 +130,9 @@ if __name__ == '__main__':
 
 4. Replace **`<YOUR_BASE_URL>`** and **`<YOUR_ACCOUNT_ID>`** with the correct values of your environment and [Access URL](/docs/cloud/about-cloud/access-regions-ip-addresses) for your region and plan.
 
-    * To find these values, navigate to **dbt Cloud**, select **Deploy -> Jobs**.  Select the Job you want to run and copy the URL. For example: `https://cloud.getdbt.com/deploy/000000/projects/111111/jobs/222222`
-    and therefore valid code would be:
+:::tip
```
Comment (matthewshaver) [inline website/docs/guides/how-to-use-databricks-workflows-to-run-dbt-cloud-jobs.md]:
```suggestion
```

File: website/docs/guides/how-to-use-databricks-workflows-to-run-dbt-cloud-jobs.md [hunk 3]
```diff
@@ -130,8 +130,8 @@ if __name__ == '__main__':
 
 4. Replace **`<YOUR_BASE_URL>`** and **`<YOUR_ACCOUNT_ID>`** with the correct values of your environment and [Access URL](/docs/cloud/about-cloud/access-regions-ip-addresses) for your region and plan.
 
-    * To find these values, navigate to **dbt Cloud**, select **Deploy -> Jobs**.  Select the Job you want to run and copy the URL. For example: `https://cloud.getdbt.com/deploy/000000/projects/111111/jobs/222222`
-    and therefore valid code would be:
+ To find these values, navigate to **dbt Cloud**, select **Deploy -> Jobs**.  Select the Job you want to run and copy the URL. For example: `https://YOUR_ACCESS_URL/deploy/000000/projects/111111/jobs/222222`
```
Comment (matthewshaver) [inline website/docs/guides/how-to-use-databricks-workflows-to-run-dbt-cloud-jobs.md]:
```suggestion
    * To find these values, navigate to **dbt Cloud**, select **Deploy -> Jobs**.  Select the Job you want to run and copy the URL. For example: `https://YOUR_ACCESS_URL/deploy/000000/projects/111111/jobs/222222`
```

File: website/docs/guides/how-to-use-databricks-workflows-to-run-dbt-cloud-jobs.md [hunk 4]
```diff
@@ -130,8 +130,8 @@ if __name__ == '__main__':
...
+and therefore valid code would be:
```
Comment (matthewshaver) [inline website/docs/guides/how-to-use-databricks-workflows-to-run-dbt-cloud-jobs.md]:
```suggestion
     and therefore valid code would be:
```

File: website/docs/guides/how-to-use-databricks-workflows-to-run-dbt-cloud-jobs.md [hunk 5]
```diff
@@ -130,8 +130,8 @@ if __name__ == '__main__':
 
 4. Replace **`<YOUR_BASE_URL>`** and **`<YOUR_ACCOUNT_ID>`** with the correct values of your environment and [Access URL](/docs/cloud/about-cloud/access-regions-ip-addresses) for your region and plan.
 
-    * To find these values, navigate to **dbt Cloud**, select **Deploy -> Jobs**.  Select the Job you want to run and copy the URL. For example: `https://cloud.getdbt.com/deploy/000000/projects/111111/jobs/222222`
-    and therefore valid code would be:
+    * To find these values, navigate to **dbt Cloud**, select **Deploy -> Jobs**.  Select the Job you want to run and copy the URL. For example: `https://YOUR_ACCESS_URL/deploy/000000/projects/111111/jobs/222222`
+     and therefore valid code would be:
```
Comment (matthewshaver) [inline website/docs/guides/how-to-use-databricks-workflows-to-run-dbt-cloud-jobs.md]:
```suggestion
    and therefore valid code would be:
```

File: website/docs/faqs/API/rotate-token.md [hunk 6]
```diff
@@ -34,23 +34,21 @@ curl --location --request POST 'https://cloud.getdbt.com/api/v3/accounts/YOUR_AC
...
-
 2. Find the new key in the API response or in dbt Cloud. 
 
 3. To find the new key in dbt Cloud, go to **Profile Settings** -> **API Access**.
 
-
 ### dbt Cloud deployments
 
-If your [dbt Cloud deployment](/docs/cloud/about-cloud/access-regions-ip-addresses) uses a different access URL, replace `cloud.getdbt.com` with the URL of your instance.
+If your [dbt Cloud deployment](/docs/cloud/about-cloud/regions-ip-addresses) uses a different access URL, replace `YOUR_ACCESS_URL` with the URL of your instance.
```
Comment (matthewshaver) [inline website/docs/faqs/API/rotate-token.md]:
```suggestion
If your [dbt Cloud deployment](/docs/cloud/about-cloud/access-regions-ip-addresses) uses a different access URL, replace `YOUR_ACCESS_URL` with the URL of your instance.
```

File: website/docs/guides/bigquery-qs.md [hunk 7]
```diff
@@ -85,7 +85,7 @@ In order to let dbt connect to your warehouse, you'll need to generate a keyfile
 3. Create a service account key for your new project from the [Service accounts page](https://console.cloud.google.com/iam-admin/serviceaccounts?walkthrough_id=iam--create-service-account-keys&start_index=1#step_index=1). For more information, refer to [Create a service account key](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating) in the Google Cloud docs. When downloading the JSON file, make sure to use a filename you can easily remember. For example, `dbt-user-creds.json`. For security reasons, dbt Labs recommends that you protect this JSON file like you would your identity credentials; for example, don't check the JSON file into your version control software.
 
 ## Connect dbt Cloud to BigQuery​
-1. Create a new project in [dbt Cloud](https://cloud.getdbt.com/). From **Account settings** (using the gear menu in the top right corner), click **+ New Project**.
+1. Create a new project in [dbt Cloud](/docs/cloud/about-cloud/regions-ip-addresses). From **Account settings** (using the gear menu in the top right corner), click **+ New Project**.
```
Comment (matthewshaver) [inline website/docs/guides/bigquery-qs.md]:
```suggestion
1. Create a new project in [dbt Cloud](/docs/cloud/about-cloud/access-regions-ip-addresses). From **Account settings** (using the gear menu in the top right corner), click **+ New Project**.
```

File: website/docs/guides/redshift-qs.md [hunk 8]
```diff
@@ -166,7 +166,7 @@ Now we are going to load our sample data into the S3 bucket that our Cloudformat
     select * from stripe.payment;
     ```
 ## Connect dbt Cloud to Redshift 
-1. Create a new project in [dbt Cloud](https://cloud.getdbt.com/). From **Account settings** (using the gear menu in the top right corner), click **+ New Project**.
+1. Create a new project in [dbt Cloud](/docs/cloud/about-cloud/regions-ip-addresses). From **Account settings** (using the gear menu in the top right corner), click **+ New Project**.
```
Comment (matthewshaver) [inline website/docs/guides/redshift-qs.md]:
```suggestion
1. Create a new project in [dbt Cloud](/docs/cloud/about-cloud/access-regions-ip-addresses). From **Account settings** (using the gear menu in the top right corner), click **+ New Project**.
```

File: website/docs/guides/starburst-galaxy-qs.md [hunk 9]
```diff
@@ -202,7 +202,7 @@ To query the Jaffle Shop data with Starburst Galaxy, you need to create tables u
     If this role is not listed for you, choose the role you selected in [Connect Starburst Galaxy to the Amazon S3 bucket](#connect-to-s3-bucket) when you added location privilege for your S3 bucket.
 3. Click **Clusters** on the left sidebar.
 4. Find your cluster in the **View clusters** table and click **Connection info**. Choose **dbt** from the **Select client** dropdown. Keep the **Connection information** modal open. You will use details from that modal in dbt Cloud.
-5. In another browser tab, log in to [dbt Cloud](https://cloud.getdbt.com/).
+5. In another browser tab, log in to [dbt Cloud](/docs/cloud/about-cloud/regions-ip-addresses).
```
Comment (matthewshaver) [inline website/docs/guides/starburst-galaxy-qs.md]:
```suggestion
5. In another browser tab, log in to [dbt Cloud](/docs/cloud/about-cloud/access-regions-ip-addresses).
```

**Comment by matthewshaver** [comment]
@mirnawong1 ready to address this one!

File: website/docs/docs/cloud/configure-cloud-cli.md [hunk 10]
```diff
@@ -40,6 +40,13 @@ Once you install the dbt Cloud CLI, you need to configure it to connect to a dbt
 
 2. Save the `dbt_cloud.yml` file in the `.dbt` directory, which stores your dbt Cloud CLI configuration. Store it in a safe place as it contains API keys. Check out the [FAQs](#faqs) to learn how to create a `.dbt` directory and move the `dbt_cloud.yml` file.
    
+    - North America: https://YOUR_ACCESS_URL/cloud-cli
```
Comment (mirnawong1) [inline website/docs/docs/cloud/configure-cloud-cli.md:43]: i know we're in beta but should we also add the azure mt? we might need to think about a snippet or easier approach to the URLs maybe for the future?

File: website/docs/guides/bigquery-qs.md [hunk 11]
```diff
@@ -85,7 +85,7 @@ In order to let dbt connect to your warehouse, you'll need to generate a keyfile
 3. Create a service account key for your new project from the [Service accounts page](https://console.cloud.google.com/iam-admin/serviceaccounts?walkthrough_id=iam--create-service-account-keys&start_index=1#step_index=1). For more information, refer to [Create a service account key](https://cloud.google.com/iam/docs/creating-managing-service-account-keys#creating) in the Google Cloud docs. When downloading the JSON file, make sure to use a filename you can easily remember. For example, `dbt-user-creds.json`. For security reasons, dbt Labs recommends that you protect this JSON file like you would your identity credentials; for example, don't check the JSON file into your version control software.
 
 ## Connect dbt Cloud to BigQuery​
-1. Create a new project in [dbt Cloud](https://cloud.getdbt.com/). From **Account settings** (using the gear menu in the top right corner), click **+ New Project**.
+1. Create a new project in [dbt Cloud](/docs/cloud/about-cloud/access-regions-ip-addresses). From **Account settings** (using the gear menu in the top right corner), click **+ New Project**.
```
Comment (mirnawong1) [inline website/docs/guides/bigquery-qs.md:88]: great catch!

**Review by mirnawong1 — APPROVED** [review]
lgtm! thank you for doing this @matthewshaver !

**Pull request merged**